scripts/partition_manager/partition_manager.rst

```diff
@@ -536,11 +536,27 @@ See :ref:`ug_bootloader_external_flash` for more details on using external flash
 
 .. _pm_build_system:
 
-You must enable device drivers for each flash partition.
-Set the ``DEFAULT_DRIVER_KCONFIG`` parameter to the Kconfig option that enables the driver during compilation.
-When a driver for a partition is not enabled, the partition will not be included in the partition manager and will not be accessible at runtime.
-You can use the :kconfig:option:`CONFIG_PM_OVERRIDE_EXTERNAL_DRIVER_CHECK` Kconfig option to override the behavior and prevent compile-time check for external flash memory devices.
-This is useful when the external device does not have a driver controlled by Kconfig options.
+For a partition to be accessible at run-time, a flash device it is placed on needs to have its driver enabled at compile time. Partition manager will not includ, into partition map, partitions that are placed on a device without a driver in the system.
+To let the partition manager know which Kconfig ensures existence of a driver, the ``DEFAULT_DRIVER_KCONFIG`` option is used.
```

Comment:
Is this an option? If its kconfig it should be prefixed with `CONFIG_` if its not kconfig, I think we should avoid calling it at "option" as that is typically used for kconfig options, and instead call it what it is (for example "cmake variable" if thats the case)

Reply:
This is param to Partition Manager script.

Reply:
Used https://github.com/nrfconnect/sdk-nrf/pull/9423#discussion_r1033976314 here